### PR TITLE
Ban !flag outside #bot-spam

### PR DESCRIPTION
**Pull request by 101arrowz**

### CODE REVIEW

**Review by henry9836 — CHANGES_REQUESTED** [review]
c

File: util/discordModules/eggModule.js [hunk 1]
```diff
@@ -221,6 +221,10 @@ function processMessage(msg, client, args){
 
     //Cris ego inflator
     else if (args[0] == "flag"){
+        if (msg.channel.id != "352299132959850497") {
+            // ban !flag outside #bot-spam
+            return true;
+        }
```
Comment (henry9836) [inline util/discordModules/eggModule.js:227]: You shouldn't use hardcoded ids, please add a setter for botConfig to add a botspam channel and utilise that under the channels section.

e.g `botConfig.channels.botspam`

**Review by henry9836 — CHANGES_REQUESTED** [review]
You shouldn't use hardcoded ids, please add a setter for botConfig to add a botspam channel and utilise that under the channels section.

e.g `botConfig.channels.botspam`

File: util/discordModules/eggModule.js (hunk 1 above)
Comment (101arrowz) [reply util/discordModules/eggModule.js:227]: fixed

**Review by henry9836 — CHANGES_REQUESTED** [review]
Looks good but the new setter command needs to be added to the admin help dialog then should be good to be approved

**Pull request merged**